torch/cuda/__init__.py

```diff
@@ -1,6 +1,12 @@
 r"""
 This package adds support for CUDA tensor types.
 
+CUDA is a parallel computing platform and programming model
+developed by NVIDIA for general computing on GPUs.
+With CUDA, developers are able to dramatically speed up computing
+applications by harnessing the power of GPUs.
+See https://developer.nvidia.com/cuda-zone for more details.
```

Comment:
Would https://en.wikipedia.org/wiki/CUDA be a better link here?

Reply:
I think so, provides more details and context.

Reply:
Added